allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -608,19 +626,26 @@ def start_test(
 
         return self.log_and_close(
             start_time_str=self.checkpoint_start_time_str(checkpoint_paths[0]),
+            checkpoint_parts=checkpoint_parts,
```

Comment:
As noted in one of my comments. I think it's cleaner to infer the output directory earlier on (e.g. in `main.py`) and rather than having to do these `checkpoints_parts` manipulations.